perceval/simulators/stepper.py

```diff
@@ -108,7 +108,7 @@ def apply(self, sv: StateVector, r: list[int], c: ACircuit) -> StateVector:
     def probs(self, input_state) -> BSDistribution:
         return _to_bsd(self.evolve(input_state))
 
-    def probs_svd(self, svd: SVDistribution, progress_callback: callable = None) -> dict:
+    def probs_svd(self, svd: SVDistribution, progress_callback: callable = None, detectors = None) -> dict:
```

Comment:
Detectors are not used. Should they also be used here so the Stepper simulator gives the same results than the Simulator?